docs/contribute/cumulative-docs/example-scenarios.md

```diff
@@ -0,0 +1,545 @@
+how it is labeled varies by whether the product/deployment mode is versioned or unversioned.
+
+For example, the majority of the content on the [Lens](https://www.elastic.co/docs/explore-analyze/visualize/lens)
+page was applicable to both Elastic Stack and to Serverless.
+One specific section describes functionality that was in technical preview in Elastic Stack 9.0.0 and
+Serverless at the time Elastic Stack 9.0.0 was released.
+Then, the functionality became generally available in Elastic Stack in 9.1.0 and shortly before the
+Elastic Stack 9.1.0 release in Serverless.
+
+* For Elastic Stack, a versioned product, label the section with both lifecycles: `ga 9.1` and `preview 9.0`.
+* For Serverless, an unversioned product, update the section label from `serverless: preview` to `serverless: ga`.
+  Do _not_ list both lifecycles.
+
+:::::{tab-set}
+::::{tab-item} Image
+:::{image} ./images/example-lifecycle-changed.png
+:screenshot:
+:alt:
+:::
+::::
+::::{tab-item} Code
+````
+---
+applies_to:
+  stack: ga
+  serverless: ga
+---
+
+# Lens [lens]
+
+[...]
+
+### Assign colors to terms [assign-colors-to-terms]
+
+```{applies_to}
+stack: ga 9.1, preview 9.0,
+serverless: ga
+```
+
+[...]
+
+````
+::::
+:::::
+
+
+## Functionality is removed [removed]
+
+When the functionality described in any level of content is removed,
+how to handle it varies by whether the product/deployment mode is versioned or unversioned.
+
+For example, we removed the `securitySolution:enableVisualizationsInFlyout` setting that was described on the
+[Configure advanced settings](https://www.elastic.co/docs/solutions/security/get-started/configure-advanced-settings)
+page from the Elastic Stack in 9.1.0 and from Serverless around the same time.
+Since this this functionality is still available before 9.1.0, we need that content to continue to be
+available to users on Elastic Stack earlier versions while communicating to users on newer versions
+that it is no longer available.
+
+:::::{tab-set}
+::::{tab-item} Image
+:::{image} ./images/example-removed-unversioned-exception.png
+:screenshot:
+:alt:
+:::
+::::
+::::{tab-item} Code
+````
+---
+applies_to:
+  stack: all
+  serverless:
+    security: all
+---
+
+# Configure advanced settings [security-advanced-settings]
+
+[...]
+
+## Access the event analyzer and Session View from the event or alert details flyout [visualizations-in-flyout]
+
+```{applies_to}
+stack: removed 9.1
+serverless: removed
+```
+
+[...]
+````
+::::
+:::::
+
+
+::::::{important}
+If the functionality was only ever available in an unversioned product or deployment mode,
+remove the content altogether.
+
+% FOR THE REVIEWER: IS THERE AN EXAMPLE?
+% I couldn't find an actual example of this scenario.
+% For example, we removed the <something> functionality that was described on the [<page>](#) page
+% from the Elastic Stack in 9.1.0.
+% Since this this functionality is still available in 9.0.0, we need that content to continue to be
```

Comment:
something "no longer true" sticking around is used on the `write` thread pool here. excuse the fucked up formatting

https://www.elastic.co/docs/reference/elasticsearch/configuration-reference/thread-pool-settings